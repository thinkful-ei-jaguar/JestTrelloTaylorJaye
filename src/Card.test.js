import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card.js';

describe('Card Component', () => {
    it('card renders successfully', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Card />, div);
        ReactDOM.unmountComponentAtNode(div);
});
    it('renders the Card html as expected', () => {
        const tree = renderer
            .create(<Card key={123} title="Burbank" content="california love"/>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });

});