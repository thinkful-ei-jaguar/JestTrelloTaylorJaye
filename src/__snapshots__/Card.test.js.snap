// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component renders the Card html as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    Burbank
  </h3>
  <p>
    california love
  </p>
</div>
`;
